habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -4,21 +4,13 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
-from typing import (
-    TYPE_CHECKING,
-    Any,
-    Dict,
-    List,
-    Optional,
-    Sequence,
-    Union,
-    cast,
-)
+from typing import Any, Dict, List, Optional, Sequence, Union, cast
 
 import numpy as np
 from gym import spaces
 from gym.spaces.box import Box
 from numpy import ndarray
+from torch import Tensor
```

Comment:
When did torch get added as a hard dependency for habitat?

Reply:
It is for CUDA enabled and noise models. I can soften.

Reply:
Yeah, soften would be good.  Don't want people thinking they have to PyTorch as their ML framework to use Habitat :)